peerUtil.go

```diff
@@ -23,7 +23,7 @@ type PeerMetrics struct {
 	MPSUp            float64
 	BPSDown          float64
 	BPSUp            float64
-	Capacity         float64
+	SendUsageRate    float64
```

Comment:
Not sure I like the name `SendUsageRate`, since this channel is for both sending and receiving. It's also not really a rate, it's the size of the backlog.

I think `FillRatio` is a good description

Reply:
Not a native speaker, but rate and ratio are just synonymous: https://www.thesaurus.com/browse/ratio?s=t. So FillRatio is perfectly fine to me too.

I just want to clarify what you mean by `this channel is for both sending and receiving`. From the code I see it's populated with the metrics from the `Peer#send` channel, which is only used when _sending_ Parcels, no? Am I misunderstand something here?

Reply:
> I just want to clarify what you mean by `this channel is for both sending and receiving`. From the code I see it's populated with the metrics from the `Peer#send` channel, which is only used when _sending_ Parcels, no? Am I misunderstand something here?

That was for changing Capacity() to SendUsageRate() for parcelchannel, which is used for both sending and receiving. I was too lazy to make it a separate PR comment, sorry!

Reply:
> Not a native speaker, but rate and ratio are just synonymous: https://www.thesaurus.com/browse/ratio?s=t. So FillRatio is perfectly fine to me too.

To me, FillRate means the rate at which the channel is being filled up (ie "100 messages a second") and FillRatio is how much the channel is filled up (ie "45% percent")

Reply:
Ok so if I understood you correctly the last commit addressed your comments.